src/routes/filter.svelte

```diff
@@ -0,0 +1,366 @@
+                <ul class="select select-list">
+                    {#each categories as category}
+                        <li
+                                on:click={(event) => updateActiveElement('category', event.target)}
+                                class="inline-button"
+                        >
+                            {category}
+                        </li>
+                    {/each}
+                </ul>
+            </div>
+        </div>
+    </div>
+
+    <div class="vacancies">
+        <div class="vacancies-header">
+            <p>What we found according to <br/> your parameters:</p>
+        </div>
+
+        <div class="vacancies-list">
+            {#each vacancies as vacancy}
+                <Vacancy data={vacancy}/>
+            {/each}
+        </div>
+    </div>
+</main>
+
+<style global>
+    header {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        height: 30vmin;
+        margin-bottom: 10vh;
+        font-family: 'Lato-bold', sans-serif;
+    }
+
+    header > img {
+        height: 70%;
+    }
+
+    header > h1 {
+        font-size: 7vmin;
+    }
+
+    .filter-container {
+        display: grid;
+        grid-template-columns: 1fr 1fr 1fr;
+        grid-column-gap: 7vh;
+    }
+
+    .filter {
+        background-color: #d5d7df;
+        border-radius: 39px;
+        margin-bottom: 7vh;
+        font-family: 'Lato';
+        transition: 5s;
+    }
+
+    .filter-header > .name {
+        padding-top: 5vh;
+        padding-left: 4vw;
+        font-family: 'Lato-bold';
+        font-size: 24pt;
+        flex: 1;
+    }
+
+    .vacancies-header {
+        font-family: 'Lato-bold';
+        font-size: 48pt;
+    }
+
+    .filter-location {
+        grid-row-start: 1;
+        grid-row-end: 3;
+    }
+
+    .filter-category {
+        grid-row-start: 1;
+        grid-row-end: 3;
+
+        grid-column-start: 3;
+        grid-column-end: 3;
+    }
+
+    .filter-content {
+        font-family: 'Lato';
+        font-size: 24pt;
+        flex: 1;
+    }
+
+    .radio-button {
+        padding-left: 4vw;
+        margin-left: 0 !important;
+        text-align: left !important;
+        border: none !important;
+    }
+
+    .radio-button:before {
+        content: '';
+        display: inline-block;
+        background-color: #fff;
+        width: 40px;
+        height: 40px;
+        margin-right: 15px;
+        border-radius: 20px;
+        top: 7px;
+        position: relative;
+        transition: all 0.2s ease;
+    }
+
+    .radio-button:hover:before {
+        background-color: #838383;
+        transition: all 0.2s ease;
+    }
+
+    .select-radio > .active:before {
+        background-color: #1364df !important;
+    }
+
+    .select {
+        padding: 0;
+        list-style-type: none;
+    }
+
+    .filter-location > .filter-content > .select,
+    .filter-category > .filter-content > .select {
+        width: 100%;
+        height: 100vh;
```

Comment:
There are gray empty spaces under the location and category filters, it is probably wiser to limit the height of the `.filter-container` selector.

Reply:
don't see it